src/test/models/users.model.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+import '../../server/models/users.model';
+
+/*
+ * Test for mongodb create.
+ */
+
+let should = require('should');
+let user = require('mongoose').model('user');
+
+describe('Users: models', function () {
+  describe('#create()', function () {
+    it('should create a new User', function (done) {
+      // Create a User object to pass to User.create()
+      var u = {
+        name: 'yoda'
+      };
+      user.create(u, function (err, createdUser) {
```

Comment:
This is still creating new users (and deleting right after) to the production database. The link I sent out earlier today should contain how to set up without having to touch the prod database. Could we add a TODO to do something like that?

An easy way to do that without much work would be 1) create a new mongoDb (instead of yoda, maybe yoda-test) as part of the test flow, 2) reset the database (drop all tables), and 3) run the tests. Then you're testing against a test db all the time. Of course, it's an overkill, but just a thought :)